gensec/customPrecon.py

```diff
@@ -62,100 +63,335 @@ def Kabsh_rmsd(atoms, initial, molindixes, removeHs=False):
-        'Ce': 3708.33, 'Pr': 3911.84, 'Nd': 3908.75, 'Pm': 3847.68, 'Sm': 3708.69, 'Eu': 3511.71,
-        'Gd': 2781.53, 'Tb': 3124.41, 'Dy': 2984.29, 'Ho': 2839.95, 'Er': 2724.12, 'Tm': 2576.78,
-        'Yb': 2387.53, 'Lu': 2371.80, 'Hf': 1274.8, 'Ta': 1019.92, 'W': 847.93, 'Re': 710.2, 'Os': 596.67,
-        'Ir': 359.1, 'Pt': 347.1, 'Au': 298.0, 'Hg': 392.0, 'Tl': 717.44, 'Pb': 697.0, 'Bi': 571.0,
-        'Po': 530.92, 'At': 457.53, 'Rn': 390.63, 'Fr': 4224.44, 'Ra': 4851.32, 'Ac': 3604.41, 'Th': 4047.54,
-        'Pa': 2367.42, 'U': 1877.10, 'Np': 2507.88, 'Pu': 2117.27, 'Am': 2110.98, 'Cm': 2403.22,
-        'Bk': 1985.82,
-        'Cf': 1891.92, 'Es': 1851.1, 'Fm': 1787.07, 'Md': 1701.0, 'No': 1578.18}
+C6_vdW = {
+    "H": 6.5000,
+    "He": 1.4600,
+    "Li": 1387.0000,
+    "Be": 214.0000,
+    "B": 99.5000,
+    "C": 46.6000,
+    "N": 24.2000,
+    "O": 15.6000,
+    "F": 9.5200,
+    "Ne": 6.3800,
+    "Na": 1556.0000,
+    "Mg": 627.0000,
+    "Al": 528.0000,
+    "Si": 305.0000,
+    "P": 185.0000,
+    "S": 134.0000,
+    "Cl": 94.6000,
+    "Ar": 64.3000,
+    "K": 3897.0000,
+    "Ca": 2221.0000,
+    "Sc": 1383.0000,
+    "Ti": 1044.0000,
+    "V": 832.0000,
+    "Cr": 602.0000,
+    "Mn": 552.0000,
+    "Fe": 482.0000,
+    "Co": 408.0000,
+    "Ni": 373.0000,
+    "Cu": 253.0000,
+    "Zn": 284.0000,
+    "Ga": 498.0000,
+    "Ge": 354.0000,
+    "As": 246.0000,
+    "Se": 210.0000,
+    "Br": 162.0000,
+    "Kr": 129.6000,
+    "Rb": 4691.0000,
+    "Sr": 3170.0000,
+    "Y": 1968.580,
+    "Zr": 1677.91,
+    "Nb": 1263.61,
+    "Mo": 1028.73,
+    "Tc": 1390.87,
+    "Ru": 609.754,
+    "Rh": 469.0,
+    "Pd": 157.5000,
+    "Ag": 339.0000,
+    "Cd": 452.0,
+    "In": 707.0460,
+    "Sn": 587.4170,
+    "Sb": 459.322,
+    "Te": 396.0,
+    "I": 385.0000,
+    "Xe": 285.9000,
+    "Cs": 6582.08,
+    "Ba": 5727.0,
+    "La": 3884.5,
+    "Ce": 3708.33,
+    "Pr": 3911.84,
+    "Nd": 3908.75,
+    "Pm": 3847.68,
+    "Sm": 3708.69,
+    "Eu": 3511.71,
+    "Gd": 2781.53,
+    "Tb": 3124.41,
+    "Dy": 2984.29,
+    "Ho": 2839.95,
+    "Er": 2724.12,
+    "Tm": 2576.78,
+    "Yb": 2387.53,
+    "Lu": 2371.80,
+    "Hf": 1274.8,
+    "Ta": 1019.92,
+    "W": 847.93,
+    "Re": 710.2,
+    "Os": 596.67,
+    "Ir": 359.1,
+    "Pt": 347.1,
+    "Au": 298.0,
+    "Hg": 392.0,
+    "Tl": 717.44,
+    "Pb": 697.0,
+    "Bi": 571.0,
+    "Po": 530.92,
+    "At": 457.53,
+    "Rn": 390.63,
+    "Fr": 4224.44,
+    "Ra": 4851.32,
+    "Ac": 3604.41,
+    "Th": 4047.54,
+    "Pa": 2367.42,
+    "U": 1877.10,
+    "Np": 2507.88,
+    "Pu": 2117.27,
+    "Am": 2110.98,
+    "Cm": 2403.22,
+    "Bk": 1985.82,
+    "Cf": 1891.92,
+    "Es": 1851.1,
+    "Fm": 1787.07,
+    "Md": 1701.0,
+    "No": 1578.18,
+}
 
 
 # Preambule from Lindh.py pthon sctipt
 HUGE = 1e10
 
-ABOHR = 0.52917721 # in AA
-HARTREE = 27.211383 # in eV
+ABOHR = 0.52917721  # in AA
```

Comment:
variables capitalized again -- and some conversions had already been defined! please make it uniform.